src/cli/evm_bytes.py

```diff
@@ -57,40 +58,57 @@ def format_assembly(self) -> str:
             return f"{opcode_name} {operands}"
 
 
-def process_evm_bytes(evm_bytes: bytes, assembly: bool = False) -> str:  # noqa: D103
+def process_evm_bytes(  # noqa: D103
+    evm_bytes: bytes, allow_unknown: bool = False
+) -> List[OpcodeWithOperands]:
     evm_bytes = bytearray(evm_bytes)
 
     opcodes: List[OpcodeWithOperands] = []
 
     while evm_bytes:
         opcode_byte = evm_bytes.pop(0)
 
-        opcode: Op
+        opcode: Opcode
         for op in Op:
             if not isinstance(op, Macro) and op.int() == opcode_byte:
                 opcode = op
                 break
         else:
-            raise ValueError(f"Unknown opcode: {opcode_byte}")
+            if allow_unknown:
+                opcode = Opcode(opcode_byte)
```

Comment:
I'm a bit skeptical of this tbh. Which tests in `ethereum/tests` require this?

Reply:
Very many (order of 20K codes). They never wrap successfully, but they fail on the `process_evm_bytes` step, where it was a bit inconvenient to handle failure. I can undo this flag and catch the `Unknown opcode` error in `eofwrap.py` instead if you prefer that.

Reply:
Did so in `Revert support for unknown opcodes in evm_bytes.py`, PTAL. I'm still running the wrapping to see if metrics and all are fine

Reply:
@marioevz If this looks good now then we can merge or do you guys have any more comments?

Then to follow-up with the including in the release - should I open an issue to track this? (I can help with this, but I'd need pointers on where to start)